### DISCUSSION TITLE
How to get the max turbo frequency of a cpu?

### DISCUSSION THREAD

**Discussion by Meldryt** [opcm/pcm#828] [Q&A]
Since i'm not able to get this information on a normal way on windows like with pdh or wmi,
i was hoping that at least intel provides a way with pcm.

F.e. for a i5-7300HQ it would be 2.50 GHz base and 3.50 GHz max turbo frequency.

**Comment by rdementi** [comment]
pcm does not directly provide this capability

**Comment by rdementi** [comment]
I see that the HWinfo tool has this feature:
![image](https://github.com/user-attachments/assets/5f1251c6-b02b-4677-b93e-aa3913a7c57b)
